package.json

```diff
@@ -22,11 +22,11 @@
     "registry": "https://registry.npmjs.org"
   },
   "dependencies": {
-    "@types/classnames": "^2.2.3",
+    "@types/classnames": "~2.2.3",
```

Comment:
A recent minor bump in `classnames` caused our imports to fail. Therefore only pull in patch bumps